style/js/shield_defs.js

```diff
@@ -559,6 +559,52 @@ export function loadShields(shieldImages) {
   shields["US:SC:Business"] = banneredShield(shields["US:SC"], ["BUS"]);
   shields["US:SC:Alternate"] = banneredShield(shields["US:SC"], ["ALT"]);
 
+  shields["US:TX"] = roundedRectShield("white", "black", "black", 1, 1, null);
+  shields["US:TX:Loop"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["LOOP"]
+  );
+  shields["US:TX:Spur"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["SPUR"]
+  );
+  shields["US:TX:Business"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["BUS"]
+  );
+  shields["US:TX:Park"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["PARK"]
+  );
+  shields["US:TX:Beltway"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["BELT"]
+  );
+  shields["US:TX:FM"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["F.M."]
+  );
+  shields["US:TX:RM"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["R.M."]
+  );
+  shields["US:TX:Recreational"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["R"]
+  );
```

Comment:
I’m unaware of any guide signs ever being posted for Recreational Roads, but how about giving it a brown outline as a nod to the reassurance marker?

<img src="https://upload.wikimedia.org/wikipedia/commons/a/a5/Texas_Recreational_Road_blank.svg" width="64" alt="R">

Reply:
I think if you're OK with distorting the state shape a whole bunch, it's achievable with a wide shield.

Reply:
Taking some liberties with the state outline, I can get a single digit route to display legibly.

We should also consider whether a banner would be necessary on this shield anyway, since it looks so similar to the Farm and Ranch--to-market reassurance markers.
![image](https://user-images.githubusercontent.com/46615954/158644437-586c64c2-9499-4120-8e40-0bdf76a21d37.png)

Reply:
That's barely workable for a 1-character shield, never mind a 5-character shield (including the space).  I think you'll need to go pretty drastic, similar to how Louisiana and Georgia are drawn in rebusurance:

https://github.com/1ec5/rebusurance/blob/main/img/canvas.png

Reply:
Even if we're turning this one into a rectangle due to constraints, I still like the idea of matching the brown of the original sign.

Reply:
Commit 8817776acf893549237e2ae7fc465bf086510e4c adds the brown outline on recreation road shields.
![image](https://user-images.githubusercontent.com/46615954/159138404-704826bc-0f03-433f-a8d6-40dc73fb077f.png)